frontend/src/metabase/components/FieldValuesWidget/FieldValuesWidget.jsx

```diff
@@ -474,18 +475,22 @@ const LoadingState = () => (
 );
 
 const NoMatchState = ({ fields }) => {
-  if (fields.length > 1) {
-    // if there is more than one field, don't name them
-    return <OptionsMessage message={t`No matching result`} />;
+  if (fields.length === 1) {
+    const [{ display_name }] = fields;
+
+    if (display_name.length <= FIELD_TITLE_MAX_LENGTH) {
```

Comment:
With i18n, I suspect there are cases where this does not work. 

How about we just remove the `display: flex` and add `text-align: center`?

![image](https://github.com/metabase/metabase/assets/6830683/b3061c6c-5321-4829-9f10-efd5398cbd18)